<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(237, 229, 218);</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>50</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>350</number>
    </property>
    <property name="topMargin">
     <number>150</number>
    </property>
    <property name="rightMargin">
     <number>350</number>
    </property>
    <property name="bottomMargin">
     <number>150</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>580</width>
        <height>560</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(41, 38, 35, 0.75);
border-radius: 20px; /* creates rounded corners */
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <property name="horizontalSpacing">
        <number>20</number>
       </property>
       <property name="verticalSpacing">
        <number>10</number>
       </property>
       <item row="1" column="0" colspan="3">
        <widget class="QLabel" name="lblCreateAccount">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>27</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;
background: transparent;
</string>
         </property>
         <property name="text">
          <string>Create an Account</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <layout class="QGridLayout" name="gridLayoutRow1">
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayoutNameLabels">
           <property name="spacing">
            <number>30</number>
           </property>
           <item>
            <widget class="QLabel" name="lblFirstName">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: white;
background: transparent;
padding-left: 10px;</string>
             </property>
             <property name="text">
              <string>First Name</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblLastName">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: white;
background: transparent;
padding-left: 10px;</string>
             </property>
             <property name="text">
              <string>Last Name</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayoutNameInputs">
           <property name="spacing">
            <number>30</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QLineEdit" name="firstNameInput">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>253</width>
               <height>37</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>100</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
color: white;
background: transparent;
border-radius: 10px; /* creates rounded corners */
border: 1px solid black;
padding-left: 5px;
background-color: rgba(41, 38, 35, 0.9);
}
QLineEdit:focus {
    border: 1px solid white;  /* border when selected */
}</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lastNameInput">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>253</width>
               <height>37</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>100</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
color: white;
background: transparent;
border-radius: 10px; /* creates rounded corners */
border: 1px solid black;
padding-left: 5px;
background-color: rgba(41, 38, 35, 0.9);
}
QLineEdit:focus {
    border: 1px solid white;  /* border when selected */
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="11" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayoutRow5">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="lblAlert">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>253</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Inter Variable Text</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: white;
background: transparent;
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnSubmit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    color: white;
    border-radius: 10px; /* creates rounded corners */
    border: 1px solid rgba(41, 38, 35, 0.9);
    padding-left: 5px;
    background-color: rgb(174, 230, 178);
}
QPushButton:pressed {
    background-color: rgb(135, 218, 141);
}
</string>
           </property>
           <property name="text">
            <string>Submit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0" colspan="3">
        <layout class="QGridLayout" name="gridLayoutRow2">
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayoutLoginLabels">
           <property name="spacing">
            <number>30</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QLabel" name="lblUserName">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: white;
background: transparent;
padding-left: 10px;
</string>
             </property>
             <property name="text">
              <string>Username</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblPassword">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: white;
background: transparent;
padding-left: 10px;
</string>
             </property>
             <property name="text">
              <string>Password</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayoutLoginInputs">
           <property name="spacing">
            <number>30</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QLineEdit" name="usernameInput">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>253</width>
               <height>37</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>100</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
color: white;
background: transparent;
border-radius: 10px; /* creates rounded corners */
border: 1px solid black;
padding-left: 5px;
background-color: rgba(41, 38, 35, 0.9);
}
QLineEdit:focus {
    border: 1px solid white;  /* border when selected */
}</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="passwordInput">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>253</width>
               <height>37</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>100</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
color: white;
background: transparent;
border-radius: 10px; /* creates rounded corners */
border: 1px solid black;
padding-left: 5px;
background-color: rgba(41, 38, 35, 0.9);
}
QLineEdit:focus {
    border: 1px solid white;  /* border when selected */
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="10" column="0" colspan="3">
        <layout class="QGridLayout" name="gridLayoutRow4" columnstretch="0">
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lblSecurityResponse">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: white;
background: transparent;
padding-left: 10px;
</string>
           </property>
           <property name="text">
            <string>Enter your response:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="securityAnswerInput">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>538</width>
             <height>37</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>80</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
color: white;
background: transparent;
border-radius: 10px; /* creates rounded corners */
border: 1px solid black;
padding-left: 5px;
background-color: rgba(41, 38, 35, 0.9);
}
QLineEdit:focus {
    border: 1px solid white;  /* border when selected */
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="0" colspan="3">
        <layout class="QGridLayout" name="gridLayoutRow3">
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="2" column="0">
          <widget class="QComboBox" name="comboBoxSecurityQuestion">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>538</width>
             <height>37</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox {
border-radius: 10px;
background-color: rgba(41, 38, 35, 0.9);
padding-left: 10px;
color: white;
border: 1px solid black;
/*font-size: 13px;*/
}
QComboBox:focus {
    border: 1px solid white;  /* border when selected */
}

/* The dropdown arrow */
QComboBox::drop-down {
	color: white;
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 10px;
}

/* The popup list */
QComboBox QAbstractItemView {
    border: 1px solid gray;
    background-color: #1e1e1e;
    selection-background-color: #444444;
    color: white;
    padding: 5px;
border-radius: 0px;
}
</string>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="maxVisibleItems">
            <number>15</number>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::NoInsert</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToMinimumContentsLength</enum>
           </property>
           <item>
            <property name="text">
             <string>What is your motherâ€™s maiden name?</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>What was the name of your first pet?</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>What city were you born in?</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>What is your Student ID?</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblSecurityQuestion">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: white;
background: transparent;
padding-left: 10px;</string>
           </property>
           <property name="text">
            <string>Select a Security Question:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
